functionary/ui/urls.py

```diff
@@ -167,6 +141,96 @@
         (scheduling.function_parameters),
         name="function-parameters",
     ),
+    path(
+        "new_schedule/",
+        (scheduling.ScheduledTaskCreateView.as_view()),
+        name="new-schedule",
+    ),
+    path(
+        "update_schedule/<uuid:pk>",
+        (scheduling.update_scheduled_task),
+        name="update-schedule",
+    ),
+    path(
+        "schedule/<uuid:pk>",
+        (scheduling.ScheduledTaskUpdateView.as_view()),
+        name="schedule-detail",
+    ),
+    path(
+        "schedule_list/",
+        (scheduling.ScheduledTaskListView.as_view()),
+        name="schedule-list",
+    ),
+]
+
+environments_urlspatterns = [
+    path(
+        "environment/<uuid:pk>",
+        (environments.EnvironmentDetailView.as_view()),
+        name="environment-detail",
+    ),
+    path(
+        "environment_list/",
+        (environments.EnvironmentListView.as_view()),
+        name="environment-list",
+    ),
+    path(
+        "environment/set_environment",
+        (environments.EnvironmentSelectView.as_view()),
+        name="set-environment",
+    ),
+    path(
+        "environment/<environment_id>/create",
+        (environments.EnvironmentCreateMemberView.as_view()),
+        name="create-environment-member",
+    ),
+    path(
+        "environment/<environment_id>/delete/<user_id>",
+        (environments.EnvironmentDeleteMemberView.as_view()),
+        name="delete-environment-member",
+    ),
+    path(
+        "environment/<environment_id>/update/<user_id>",
+        (environments.EnvironmentUpdateMemberView.as_view()),
+        name="update-environment-member",
+    ),
 ]
 
-urlpatterns += htmx_urlpatterns
+teams_urlpatterns = [
+    path("team_list/", (teams.TeamListView.as_view()), name="team-list"),
+    path(
+        "team/<uuid:pk>",
+        (teams.TeamDetailView.as_view()),
+        name="team-detail",
+    ),
+    path(
+        "team/<team_id>/create",
+        (teams.TeamCreateMemberView.as_view()),
+        name="create-team-member",
+    ),
+    path(
+        "team/<team_id>/delete/<user_id>",
+        (teams.TeamDeleteMemberView.as_view()),
+        name="delete-team-member",
+    ),
+    path(
+        "team/<team_id>/update/<user_id>",
+        (teams.TeamUpdateMemberView.as_view()),
+        name="update-team-member",
+    ),
+    path("users/", (teams.get_users), name="get-users"),
+]
+
+
+"""
+Append App URL patterns to a main urlpatterns list.
+
+URL patterns should be grouped by the django app they represent.
+
+For example, there should be a list for all the URLs related to
+teams, environments, schedules, tasks, etc.
+
+"""
+urlpatterns += environments_urlspatterns
+urlpatterns += schedules_urlpatterns
+urlpatterns += teams_urlpatterns
```

Comment:
Nit - I would make env/sched/team singular and leave urlpatterns plural. Also need to remove the extra s in environments_url**s**patterns

Reply:
Fixed